ladybug/dt.py

```diff
@@ -256,23 +271,26 @@ def __repr__(self):
 class Date(date):
     """Ladybug Date.
 
+    Args:
+        month: A value for month between 1-12 (Defualt: 1).
+        day: A value for day between 1-31 (Defualt: 1).
+        leap_year: A boolean to indicate if date is for a leap year
+            (Default: False).
```

Comment:
Let's remove the parentheses around the Default and just add a period after `year`. This is the new standard we are trying to follow throughout the libraries.